test/trackler/implementation_test.rb

```diff
@@ -89,6 +89,15 @@ def test_readme_has_empty_string_for_track_hint_when_setup_file_does_not_exist
     assert_equal expected, implementation.readme
   end
 
+  def test_readme_uses_track_hint_instead_of_setup
```

Comment:
This doesn't actually check the 'instead_of_setup' bit.
`test_readme_users_track_hint_file` is probably enough of a description.